packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -52,6 +52,11 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
     )..addListener(markNeedsPaint);
   }
 
+  /// Whether this control is current "active" or "inactive".
```

Comment:
for clarification maybe say something like `"active" (checked, on, selected) or "inactive" (unchecked, off, not selected)`